src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx

```diff
@@ -30,6 +30,41 @@ const App = () => (
 );
 ```
 
+2. Render two `MeetingProvider`s near the root of your application with the same `MeetingManager` instance.
+
+```jsx
+import { MeetingProvider, useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+
+const Root = () => {
+  const meetingManager = new MeetingManager({ logLevel: LogLevel.INFO });
+  
+  return (
+    <MeetingProvider meetingManager={meetingManager}>
+      <MyApp1 />
+    <MeetingProvider/>
+
+    <MeetingProvider meetingManager={meetingManager}>
+      <MyApp2 />
+    <MeetingProvider/>
+  ); 
+}
+
+const MyApp1 = () => {
+  const meetingManager = useMeetingManager();
+
+  useEffect(() => {
+    meetingManager.getAttendee = async (chimeAttendeeId: string, externalUserId?: string) => {
+      const response = await fetch('/my-attendees-endpoint);
+      const user = await response.json();
+
+      return {
+        name: user.name
+      }
+    }
+  }, [])
```

Comment:
I would remove this as it's not specific to the example of using two meeting managers

Reply:
Cool, removed it. Thanks.